src/hyperion/utils/lookup_table.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC, abstractmethod
+
+import numpy as np
+from numpy import interp, loadtxt
+
+from hyperion.log import LOGGER
+
+
+class LookupTableConverter(ABC):
+    """Interface for generic lookup table functionality."""
+
+    @abstractmethod
+    def s_to_t(self, s):
+        pass
```

Comment:
Could: As above it's not clear what other things will use this interface and so I would not write it yet.